gems/sorbet-static-and-runtime/sorbet-static-and-runtime.gemspec

```diff
@@ -0,0 +1,18 @@
+Gem::Specification.new do |s|
+  s.name        = 'sorbet-static-and-runtime'
+  s.version     = '0.0.0'
+  s.summary     = 'A Typechecker for Ruby'
+  s.description = 'Sorbet static and runtime in one gem'
+  s.authors     = ['Stripe']
+  s.email       = 'sorbet@stripe.com'
+  s.homepage    = 'https://sorbet.run'
+  s.license     = 'Apache-2.0'
+  s.metadata = {
+    'source_code_uri' => 'https://github.com/sorbet/sorbet'
+  }
+
+  s.add_dependency 'sorbet', '0.0.0'
+  s.add_dependency 'sorbet-runtime', '0.0.0'
+
+  s.required_ruby_version = ['>= 2.3.0']
+end
```

Comment:
This gem is not going to be automatically built. You will need to find somewhere to run `gem build`, and also do the logic to `sed` in the version.

Reply:
I added a `.buildkite/build-sorbet-static-and-runtime.sh` script to call `gem build` and apply the `sed`.

I also added an entry for it in the `.buildkite/pipeline.yaml`.

Reply:
The result can be seen here: https://buildkite.com/sorbet/sorbet/builds/19757#4c991446-3857-4711-9f79-488725fd6ca1